src/store/ChartStore.js

```diff
@@ -224,6 +224,20 @@ class ChartStore {
         this.holderStyle = stxx.chart.panel.holder.style;
 
         stxx.append('deleteHighlighted', this.updateComparisons);
+
+        stxx.append('adjustPanelPositions', function () {
+            let index = 0;
+            for (const x in stxx.panels) {
```

Comment:
@MahboobehMohammadi also it's work fine. but think it's better to prevent loop in each `adjustPanelPositions`, Could you please handle it in another way?

Reply:
@shayanbinary this loop only happen twice, for index 0 and 1, and  to find the panel inside the stx.pannels, i need panel name and dont have it here, its why im using this for loop, actually its the way chartiq.js itself access panels.
http://documentation.chartiq.com/CIQ.ChartEngine.Panel.html#Panel

Reply:
@MahboobehMohammadi all look good to me. I am just thinking about something like `const Index = Object.keys(stxx.panels)[1];const indicate = sttx.panels[index];indicate.up.style.display = 'none';`. just to use less code.

Reply:
@shayanbinary you got it